gem/lib/metasploit-payloads/crypto.rb

```diff
@@ -0,0 +1,44 @@
+require 'openssl'
+
+module MetasploitPayloads
+  module Crypto
+    CIPHER_NAME = 'chacha20'.b.freeze
```

Comment:
https://stackoverflow.com/questions/2873903/how-to-i-make-private-class-constants-in-ruby

```
private_costant :CIPHER_NAME
```

So it's not part of our public API